TwilioVerifySDK/TwilioSecurity/Sources/Storage/AuthenticatedSecureStorage.swift

```diff
@@ -130,12 +134,12 @@ extension AuthenticatedSecureStorage: AuthenticatedSecureStorageProvider {
     }
   }
 
-  private func evaluatePolicy(for authenticator: Authenticator, success: @escaping EmptySuccessBlock, failure: @escaping ErrorBlock) {
+  private func evaluatePolicy(for authenticator: Authenticator, withDeviceOwnerAuthentication: Bool = false, success: @escaping EmptySuccessBlock, failure: @escaping ErrorBlock) {
```

Comment:
I would prefer two different methods: `canEvaluatePolicy` and `evaluatePolicy`. `evaluatePolicy` can call `canEvaluatePolicy` before calling the context. The boolean could be confusing

Reply:
Sounds Good, I'm gonna make the change. Thank you.